packages/ui/src/components/navbar-new/app-sidebar.tsx

```diff
@@ -0,0 +1,219 @@
+import { ThemeDialog } from '@components/theme-selector-v2'
+
+import { SidebarItem } from './sidebar-item'
+import { SidebarSearch } from './sidebar-search'
+import { SearchProvider } from './sidebar-search-new/search-context'
+import { SidebarSearchNew } from './sidebar-search-new/sidebar-search-new'
+import { User } from './sidebar-user'
+import { NavbarItemType } from './types'
+
+interface SidebarProps {
+  recentMenuItems: NavbarItemType[]
+  pinnedMenuItems: NavbarItemType[]
+  showMoreMenu: boolean
+  showSettingMenu: boolean
+  handleMoreMenu: () => void
+  handleSettingsMenu: () => void
+  currentUser: TypesUser | undefined
+  handleCustomNav: () => void
+  handleLogOut: () => void
+  handleChangePinnedMenuItem: (item: NavbarItemType, pin: boolean) => void
+  handleRemoveRecentMenuItem: (item: NavbarItemType) => void
+  useThemeStore: () => IThemeStore
+  useTranslationStore: () => TranslationStore
+  showNewSearch?: boolean
+}
+
+export const AppSidebar = ({
+  useThemeStore,
+  useTranslationStore,
+  handleChangePinnedMenuItem,
+  handleRemoveRecentMenuItem,
+  pinnedMenuItems,
+  recentMenuItems,
+  currentUser,
+  handleMoreMenu,
+  handleSettingsMenu,
+  handleCustomNav,
+  handleLogOut,
+  showNewSearch
+}: SidebarProps) => {
+  const { t, i18n, changeLanguage } = useTranslationStore()
+  const { theme, setTheme } = useThemeStore()
+  const navigate = useNavigate()
+
+  const [openThemeDialog, setOpenThemeDialog] = useState(false)
+  const [openLanguageDialog, setOpenLanguageDialog] = useState(false)
+
+  const handleLanguageChange = (language: LanguageInterface) => {
+    changeLanguage(language.code.toLowerCase())
+  }
+
+  const handleLanguageSave = (language: LanguageInterface) => {
+    changeLanguage(language.code.toLowerCase())
+    setOpenLanguageDialog(false)
+  }
+
+  const handleLanguageCancel = () => {
+    setOpenLanguageDialog(false)
+  }
+
+  return (
+    <>
+      <Sidebar.Root>
+        <Sidebar.Header>
+          {showNewSearch ? (
+            <SearchProvider>
+              <SidebarSearchNew
+                logo={
+                  <Link to="/" className="h-[58px] flex gap-2 items-center pl-3 justify-start">
+                    <Icon name="harness" size={20} className="text-foreground-accent" />
+                    <Icon name="harness-logo-text" size={68} className="text-foreground-1" />
+                  </Link>
+                }
+              />
+            </SearchProvider>
+          ) : (
+            <SidebarSearch
+              logo={
+                <Link className="flex items-center gap-1.5" to="/">
+                  <Icon name="harness" size={18} className="text-foreground-accent" />
+                  <Icon name="harness-logo-text" width={65} height={15} className="mb-0.5 text-foreground-1" />
+                </Link>
+              }
+            />
+          )}
+        </Sidebar.Header>
+        <Sidebar.Content>
+          <Sidebar.Group className="px-4 pt-5">
```

Comment:
```suggestion
          <Sidebar.Group className="px-2 pt-5">
```
<img width="789" alt="image" src="https://github.com/user-attachments/assets/b2e86f75-3ea1-4992-b8d9-9a7f89213198" />